### ISSUE TITLE
For Debian 7 and Debian 8 platforms，install OMI package failed with omi-1.3.0-0 version

### ISSUE THREAD

**Issue by Wayenyin** [Microsoft/omi#342]
For Debian 7 and Debian 8 platforms(both x86 and x64), when we installed OMI package, it would fail, the details were as following:
```
root@omi86-deb7-01  # dpkg -i omi-1.3.0-0.ssl_100.ulinux.x86.deb 
Selecting previously unselected package omi.
(Reading database ... 25033 files and directories currently installed.)
Unpacking omi (from omi-1.3.0-0.ssl_100.ulinux.x86.deb) ...
Creating omiusers group ...
Setting up omi (1.3.0.0) ...
Generating a 2048 bit RSA private key
.........................................................................+++
..........................................................+++
writing new private key to '/etc/opt/omi/ssl/omikey.pem'
-----
Configuring OMI service ...
/var/lib/dpkg/info/omi.postinst: 523: /var/lib/dpkg/info/omi.postinst: initctl: not found
omid: unrecognized service
```
**Expect** : Successfully install OMI package and omi service started.
**Actual** : The installation would pop up errors and the omi service didn't start.

**Build version:**
   omi-1.3.0-0

**Comment by yakman2020** [comment]
@Microsoft/omi-devs  This has been fixed in build 1.3.0-1

**Comment by Wayenyin** [comment]
We have tested this issue with OMI version 1.3.0-1, and it works fine on Debian 7 and Debian 8 platforms(both x86 and x64).
**E.g Debian 8 x86**
```
root@omi86-deb8-01  # dpkg -i omi-1.3.0-1.ssl_100.ulinux.x86.deb 
Selecting previously unselected package omi.
(Reading database ... 30341 files and directories currently installed.)
Preparing to unpack omi-1.3.0-1.ssl_100.ulinux.x86.deb ...
Creating omiusers group ...
Unpacking omi (1.3.0.1) ...
Setting up omi (1.3.0.1) ...
Generating a 2048 bit RSA private key
..................+++
........+++
writing new private key to '/etc/opt/omi/ssl/omikey.pem'
-----
Configuring OMI service ...
Processing triggers for systemd (215-17) ...

~
root@omi86-deb8-01  # dpkg -l|grep omi
ii  omi                           1.3.0.1                     i386         Open Management Infrastructure
~
root@omi86-deb8-01  # ps -ef|grep omi
root     15388     1  0 18:13 ?        00:00:00 /opt/omi/bin/omiserver --configfile=/etc/opt/omi/conf/omiserver.conf -d
root     15405   918  0 18:15 pts/0    00:00:00 grep omi
```

**Build version:**
omi-1.3.0-1
